Frontend/src/app/players-list/players-list.component.html

```diff
@@ -0,0 +1,25 @@
+<table *ngIf="players != null && players.length > 0">
+	<tr>
+		<th>Nome</th>
+		<th>Ações</th>
+	</tr>
+	<tr *ngFor="let player of players; let i = index">
+		<td>{{player.nickname}}</td>
+		<td>
+			<span (click)="updateUser(i)">Atualizar</span>
+			<span (click)="deleteUser(i)">Remover</span>
+		</td>
+	</tr>
+</table>
+
+<p *ngIf="players == null || players.length == 0">Nenhum jogador encontrado</p>
```

Comment:
No need to fix, but when I have more 'complicated' ngIf expression, I like to create a method (or a getter) for it. This is useful in the 'else', where you actually have to negate the whole expression. In this case, I'd do something like:

```
private hasPlayers() {
  return players != null && players.length > 0;
}
```
Your template would be like:
```
<table *ngIf="hasPlayers()">
...
<p *ngIf="!hasPlayers()">Nenhum jogador encontrado</p>
```

A slightly more 'elegant' solution, would be a getter instead of a regular function:

```
// Notice the 'get' keyword before the function name.
private get hasPlayers() {
  return players != null && players.length > 0;
}
```

Your template would be like:
```
// Notice the missing '()'. This is because a getter creates a 'virtual property'.
// It's looks like you have a property called hasPlayers, but it actually runs the function every time you try to read it.
<table *ngIf="hasPlayers">
...
<p *ngIf="!hasPlayers">Nenhum jogador encontrado</p>
```